### PR TITLE
modified create ai pipeline doc

### PR DESCRIPTION
**Pull request by oindrillac**
## Related Issues and Dependencies

https://github.com/aicoe-aiops/meteor-demo/issues/3

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by oindrillac** [comment]
cc : @isabelizimm @MichaelClifford @chauhankaranraj @pacospace

File: docs/source/create-ai-pipeline.md [hunk 1]
```diff
@@ -47,16 +47,16 @@ If you are more familiar with using CLI from the terminal you can use the follow
 2. Run the following command with your specific inputs:
 
 ```bash
-elyra-metadata install runtime-images --display_name="Tutorial Training Step" --description="Training Step Tutorial" --image_name="quay.io/thoth-station/elyra-aidevsecops-training:v0.5.0"
+elyra-metadata install runtime-images --display_name="Tutorial Training Step" --description="Training Step Tutorial" --image_name="quay.io/thoth-station/elyra-aidevsecops-training:v0.10.0"
 ```
 
-If you want to know more, check the following [link](https://elyra.readthedocs.io/en/v2.0.1/user_guide/runtime-image-conf.html).
+To learn more adding runntime images, check the following [link](https://elyra.readthedocs.io/en/v2.0.1/user_guide/runtime-image-conf.html).
```
Comment (MichaelClifford) [inline docs/source/create-ai-pipeline.md:53]:
```suggestion
To learn more about adding runtime images, check the following [link](https://elyra.readthedocs.io/en/v2.0.1/user_guide/runtime-image-conf.html).
```

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

File: docs/source/create-ai-pipeline.md (hunk 1 above)
Comment (oindrillac) [reply docs/source/create-ai-pipeline.md]: changed :)

**Pull request merged**